.github/workflows/release-crx.yml

```
@@ -0,0 +1,209 @@
+
+      - name: Resolve release context
+        id: resolve_tag
+        env:
+          EVENT_NAME: ${{ github.event_name }}
+          RELEASE_TAG: ${{ github.event.release.tag_name }}
+          IS_PRERELEASE: ${{ github.event.release.prerelease }}
+          DISPATCH_UPLOAD: ${{ github.event.inputs.upload_to_release }}
+          DISPATCH_CHANNEL_TAG: ${{ github.event.inputs.channel_tag }}
+          CHECKOUT_REF: ${{ github.event_name == 'release' && github.event.release.tag_name || github.event.inputs.ref || github.event.repository.default_branch }}
+          PACKAGE_VERSION: ${{ steps.package.outputs.package-version }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:25:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)